server/src/test/java/io/unitycatalog/server/base/catalog/BaseCatalogCRUDTest.java

```diff
@@ -1,37 +1,39 @@
 package io.unitycatalog.server.base.catalog;
 
 import io.unitycatalog.client.ApiException;
+import io.unitycatalog.client.model.CatalogInfo;
 import io.unitycatalog.client.model.CreateCatalog;
 import io.unitycatalog.client.model.UpdateCatalog;
 import io.unitycatalog.server.base.BaseCRUDTest;
-import io.unitycatalog.server.persist.PropertyRepository;
-import org.junit.*;
-
-import io.unitycatalog.client.model.CatalogInfo;
+import org.junit.Test;
 
 import java.util.List;
+import java.util.Map;
 import java.util.Objects;
 import java.util.Optional;
 
 import static io.unitycatalog.server.utils.TestUtils.*;
+import static org.assertj.core.api.Assertions.assertThat;
 
 public abstract class BaseCatalogCRUDTest extends BaseCRUDTest {
 
-    protected void assertCatalog(CatalogInfo catalogInfo, String name, String comment) {
-        Assert.assertEquals(name, catalogInfo.getName());
-        Assert.assertEquals(comment, catalogInfo.getComment());
-        Assert.assertNotNull(catalogInfo.getCreatedAt());
+    protected void assertCatalog(CatalogInfo catalogInfo, String name, String comment, Map<String, String> properties) {
+        assertThat(catalogInfo.getName()).isEqualTo(name);
+        assertThat(catalogInfo.getComment()).isEqualTo(comment);
+        assertThat(catalogInfo.getProperties()).isEqualTo(properties);
```

Comment:
Let us not add this assert right now so that the tests do not fail, we will fix it with the TODO on line 25

Reply:
@vikrantpuppala thanks so much for reviewing, but it seems that pending the properties assertion right now can't prevent cli-related errors